sites/example-project/src/components/viz/Value.svelte

```diff
@@ -85,7 +85,7 @@
 	<span class="error">
 		<span class="error-label">Error</span>
 		<span class="additional-info-icon">
-			<HelpCircleIcon height="18" width="18" verticalOffset="2" color="--grey-000" />
+			<HelpCircleIcon height="18" width="18" verticalOffset="2" color="--grey-100" />
```

Comment:
Not 100% sure what's going on here, but I'm not seeing the icon in Safari

## Chrome
<img width="591" alt="CleanShot 2023-05-15 at 08 22 50@2x" src="https://github.com/evidence-dev/evidence/assets/58074498/cb76ddc7-2f41-4458-89df-08ddb8d37758">


## Safari
<img width="627" alt="CleanShot 2023-05-15 at 08 20 21@2x" src="https://github.com/evidence-dev/evidence/assets/58074498/8acb4916-4f13-48bc-8af5-f28020832079">

Reply:
Thanks for the review! :)
The cause might be `display: inline`. Without explicitly setting the dimensions, it was probably misrendering SVG in Safari. Fixed that.